react/components/FooterLinkList.js

```diff
@@ -4,23 +4,25 @@ import React from 'react'
 import footerList from './footerList'
 import footer from '../footer.css'
 
-
-export const FooterLinkItem = ({ url, title, target }) => (
-  <a
-    className={`${footer.listLink} dib no-underline underline-hover mb0 mt2 w-100 t-small truncate c-muted-1 pointer`}
-    href={url}
-    target={target ? target : '_self'}
-  >
-    {title}
-  </a>
-)
+export const FooterLinkItem = ({ url, title, target }) =>
+  title ? (
+    <a
+      className={`${
+        footer.listLink
```

Comment:
You could use `classnames` instead of string interpolation, and then remove the tokens from `footer.css` that doesn't have any css rule.

Reply:
I'll fix in another PR.